docs-src/socket-mode/index.rst

```diff
@@ -47,6 +62,44 @@ You will be using the app-level token that starts with ``xapp-`` prefix. Note th
                     channel=req.payload["event"]["channel"],
                     timestamp=req.payload["event"]["ts"],
                 )
+        if req.type == "interactive" \
+            and req.payload["type"] == "shortcut":
```

Comment:
The type can be None
```suggestion
            and req.payload.get("type") == "shortcut":
```

Reply:
Ah, never mind. As long as `req.type == "interactive"` is checked beforehand, accessing `req.payload["type"]` should be safe

Reply:
oop, I just pushed the changes for `.get` 
I'm fine with leaving it `.get` incase people decide to later copy/paste from it since it should be safe if other payload types might not have the same guarantee

Reply:
yeah, either is fine!